dart/dynamics/FreeJoint.cpp

```diff
@@ -508,25 +508,9 @@ void FreeJoint::setAngularAcceleration(
 
 //==============================================================================
 Eigen::Matrix6d FreeJoint::getLocalJacobianStatic(
-    const Eigen::Vector6d& _positions) const
+    const Eigen::Vector6d& /*positions*/) const
```

Comment:
Oh wait, I take that back. Does this function imply that the Jacobian is now completely agnostic to the current position values?